Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+$color: #b2db59;
+
+.tech {
+  align-items: center;
+  background-color: $color;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  margin: 20px;
+  min-height: 280px;
+  padding: 6px;
+  position: relative;
+  width: 200px;
+
+  &::before {
+    bottom: 10px;
+    border: 1px solid $color;
+    border-radius: 5px;
+    content: '';
+    left: -10px;
+    position: absolute;
+    right: 10px;
+    top: -10px;
+    z-index: -1;
+  }
+
+  .tech-img {
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 70px;
+    width: 80px;
+  }
+
+  .tech-title {
+    margin-top: -40px;
```

Comment:
Avoid negative margins at all costs. I think you dont need this, why are you using it?